downstream/modules/platform/proc-operator-external-db-controller.adoc

```diff
@@ -6,7 +6,14 @@
 [role="_abstract"]
 For users who prefer to deploy {PlatformNameShort} with an external database, they can do so by configuring a secret with instance credentials and connection information, then applying it to their cluster using the `oc create` command.
 
-By default, the {PlatformName} operator automatically creates and configures a managed PostgreSQL pod in the same namespace as your {PlatformNameShort} deployment. A user may instead choose to use an external database if they prefer to use a dedicated node to ensure dedicated resources or to manually manage backups, upgrades, or performance tweaks. The following section outlines the steps to configure an external database for your {ControllerName} on a {PlatformNameShort} operator.
+By default, the {PlatformName} operator automatically creates and configures a managed PostgreSQL pod in the same namespace as your {PlatformNameShort} deployment. A user may instead choose to use an external database if they prefer to use a dedicated node to ensure dedicated resources or to manually manage backups, upgrades, or performance tweaks.
```

Comment:
The second sentence "A user may instead" sounds really confusing to me. I would advocate for addressing the reader directly so change "A user" to "You".

Also what is meant by "a dedicated node to ensure dedicated resources"? It seems like these two sentences could be collapsed a little. You could probably even condense this with the first sentence. That seems like it is slightly redundant with the "A user may" sentence, which addresses motivation.

Perhaps consider something like the following:

You can deploy {PlatformNameShort} with an external database instead of the managed PostgreSQL pod that the {PlatformName} operator automatically creates.
Using an external database lets you share and reuse resources and manually manage backups, upgrades, and performance optimizations.

Reply:
Thanks for the feedback @oraNod, I have implemented your suggested changes.